app/src/main/java/com/duckduckgo/app/tabs/ui/TabSwitcherActivity.kt

```diff
@@ -207,6 +211,7 @@ class TabSwitcherActivity : DuckDuckGoActivity(), TabSwitcherListener, Coroutine
             settingsDataStore = settingsDataStore,
             userEventsStore = userEventsStore,
             appCoroutineScope = appCoroutineScope,
+            featureSegmentsManager = featureSegmentsManager,
```

Comment:
is this needed?

Reply:
yes, we create an instance of the `FireDialog` here which needs the `featureSegmentsManager` as an argument